test-case/manual/jack_detection_boot.md

```diff
@@ -0,0 +1,60 @@
+# Jack detection for headset/HDMI/DP during boot test
+Jack detection for headset/HDMI/DP during boot is part of a series of manual
+tests.
+
+# Preconditions
+1. System is powered off.
+
+# Test Description
+* Plug headset into unit, and determine if status is on as expected after power
+on and off after power down and removal.
```

Comment:
What do you mean here by 
```
after power on and off after power down and removal.
```

I think we can only check the Jack status after power on?

Reply:
Ha yeah, I definitely worded that weird.